src/Components/FeelingRoom/FeelingRoom.scss

```diff
@@ -23,6 +25,8 @@
   background-color: hsla(199, 14%, 74%, .7);
   height: 14vh;
   width: 12vw;
+  position: fixed;
```

Comment:
should we change the z-index of the footer as well so that it sits on top of each animation?

Reply:
I think that would be great, since our user can't click on the door during some of our animations currently.